experiments/modern/src/App.js

```diff
@@ -49,11 +59,18 @@ async function getSkin() {
     // TODO: This is probalby only valid if in an `<elements>` node
     switch (node.name) {
       case "bitmap": {
-        const { file, gammagroup, h, id, w, x, y } = node.attributes;
+        let { file, gammagroup, h, id, w, x, y } = node.attributes;
         // TODO: Escape file for regex
         const img = Utils.getCaseInsensitveFile(zip, file);
         const imgBlob = await img.async("blob");
         const imgUrl = URL.createObjectURL(imgBlob);
+        if (w  === undefined || h  === undefined) {
+          let [width, height] = await loadImage(imgUrl);
```

Comment:
I bet this will bust the debugger 😔

Reply:
we already have an `await` above it, so was hoping it didn't change anything else :-X

Reply:
Oh, I lost track of where this was. This isn’t part of the interpreter/debugger. Sorry!